include/fastdds/dds/xtypes/dynamic_types/DynamicTypeBuilderFactory.hpp

```diff
@@ -117,7 +117,7 @@ class DynamicTypeBuilderFactory : public std::enable_shared_from_this<DynamicTyp
     /*!
      * Creates a new @ref DynamicTypeBuilder reference representing an array.
      * @param[in] element_type @ref DynamicType reference which becomes the element type
-     * @param[in] bounds `uint32_t` representing the desired dimensions.
+     * @param[in] bound `uint32_t` representing the desired dimensions.
```

Comment:
```suggestion
     * @param[in] bound `uint32_t` sequence representing the desired dimensions.
```

Reply:
Fixed in [8020bf2](https://github.com/eProsima/Fast-DDS/pull/4240/commits/8020bf2136b0ed793a5e093b26fc59595dbd7b26)